script.js

```diff
@@ -0,0 +1,127 @@
+// HTML Elements
+const statusDiv = document.querySelector('.status');
+const resetDiv = document.querySelector('.reset');
+const cellDivs = document.querySelectorAll('.game-cell');
+
+// game constants
+const xSymbol = '×';
+const oSymbol = '○';
+
+// game variables
+let gameIsLive = true;
+let xIsNext = true;
+
+
+// functions
+const letterToSymbol = (letter) => letter === 'x' ? xSymbol : oSymbol;
+
+const handleWin = (letter) => {
+  gameIsLive = false;
+  if (letter === 'x') {
+    statusDiv.innerHTML = `${letterToSymbol(letter)} has won!`;
+  } else {
+    statusDiv.innerHTML = `<span>${letterToSymbol(letter)} has won!</span>`;
```

Comment:
innerHTML presents some security vulnerabilities and is generally considered unsafe. innerText is safe though! if you want to put html elements into your DOM, use the 3-step process that we learned in intro-to-fetch